.gitmodules

```diff
@@ -1,9 +1,9 @@
-[submodule "oneTBB"]
```

Comment:
if someone only does git clone without parameters, no submodules will be cloned. It only happens when recursive parameters are added.

Reply:
Compiling issue should not exist anymore. The submodules are only used for webassembly

Reply:
If git clone is used without using recursive parameters, the submodules will not be be cloned.Even it is cloned, it will not be used by regular compiling.